naptime-graphql/src/main/scala/org/coursera/naptime/ari/graphql/GraphqlSchemaProvider.scala

```diff
@@ -73,7 +74,10 @@ class DefaultGraphqlSchemaProvider @Inject() (schemaProvider: SchemaProvider)
       }
     }.toMap
     try {
-      val builder = new SangriaGraphQlSchemaBuilder(latestSchema.resources, types)
+      val resources = latestSchema.resources.filter { resource =>
```

Comment:
@bryan-coursera I'm having second thoughts about filtering the schema in file vs maybe the `GraphQLController`. It seems sort of gimmicky to me to have an option "filterOnlyMultigets" which is why I placed inside the schema provider, but if you think otherwise, let me know.